src/schoology-extractor/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "python.linting.pylintEnabled": true,
+  "python.pythonPath": ".venv\\Scripts\\python.exe"
+}
```

Comment:
I committed this file by error...  Do you still want to keep it?

Reply:
If setting the python path there is helpful, then keep the file.

Reply:
It could be helpful but only if you open vsCode with the src/schoology-extractor as the root folder. I think I'll keep it with the required modifications.